src/pytorch_lightning/strategies/tpu_spawn.py

```diff
@@ -169,11 +174,11 @@ def configure_ddp(self) -> None:
     def model_to_device(self) -> None:
         self.model = self.wrapped_model.to(self.root_device)
 
-    def barrier(self, name: Optional[str] = None) -> None:
+    def barrier(self, name: Optional[str] = None, *args: Any, **kwargs: Any) -> None:
         if self.is_distributed:
             rendezvous(name)
 
-    def broadcast(self, obj: object, src: int = 0) -> object:
+    def broadcast(self, obj: object, src: int = 0) -> Any:
```

Comment:
Shouldn't `obj` and the return type match?

```suggestion
    def broadcast(self, obj: Any, src: int = 0) -> Any:
```

Reply:
Yes, we should change it to TBroadcast like in the other strategies

Reply:
I updated it to TBroadcast. https://github.com/Lightning-AI/lightning/pull/13813/commits/ffbd23cf96abcaa910501217e48d52b53893536d